### PR TITLE
Added health checks

### PR DESCRIPTION
**Pull request by suyashgautam**

### CODE REVIEW

File: apps/api/src/app/app.module.ts [hunk 1]
```diff
@@ -16,14 +19,13 @@ import { PosthogModule } from 'nestjs-posthog';
...
-        return {
-          name: 'db',
-          url: config.get('REDIS_URI'),
-        };
-      },
-      inject: [ConfigService],
+    RedisModule.forRoot({
+      readyLog: true,
+      config: {
+          host: 'localhost',
```
Comment (radhay-samagra) [inline apps/api/src/app/app.module.ts:25]: Hey @suyashgautam , can you check the previous dependency injection method used. Its not good practise to expose env variables here. We need to fetch it from .env file.

**Review by radhay-samagra — COMMENTED** [review]
Remove hard coded env variables from app module

File: apps/api/src/app/app.module.ts (hunk 1 above)
Comment (suyashgautam) [reply apps/api/src/app/app.module.ts]: Hey @radhay-samagra , I have fixed the hard coded env variables issue. Do let me know if there is anything more I can improve.

**Comment by suyashgautam** [comment]
Sure here is the success response
[Successful.txt](https://github.com/Samagra-Development/yaus/files/8935612/Successful.txt)

**Comment by radhay-samagra** [comment]
LGTM!! @ChakshuGautam

**Pull request merged**